### PR TITLE
Expose header of response

### PR DESCRIPTION
**Pull request by tzvatot**
@jhernand @zgalor PTAL

### CODE REVIEW

File: pkg/client/response.go [hunk 1]
```diff
@@ -44,3 +44,8 @@ func (r *Response) Bytes() []byte {
 func (r *Response) String() string {
 	return string(r.body)
 }
+
+// Header returns the header value.
+func (r *Response) Header(name string) string {
+	return r.header.Get(name)
```
Comment (zgalor) [inline pkg/client/response.go:53]: Are you sure header is not nil at this point?

File: pkg/client/response.go [hunk 2]
```diff
@@ -44,3 +44,8 @@ func (r *Response) Bytes() []byte {
 func (r *Response) String() string {
 	return string(r.body)
 }
+
+// Header returns the header value.
```
Comment (jhernand) [inline pkg/client/response.go]: What will this return if there is no such header? Answer that in the documentation of the method.

Comment (zgalor) [reply pkg/client/response.go]: the `Get` function documentation says an empty string will be returned in such case

Comment (tzvatot) [reply pkg/client/response.go]: I'll add it to the documentation

File: pkg/client/response.go (hunk 1 above)
Comment (tzvatot) [reply pkg/client/response.go:53]: Added a check to be safe.

**Comment by zgalor** [comment]
LGTM

**Review by jhernand — APPROVED** [review]

**Pull request merged**